object/types.proto

```diff
@@ -12,34 +12,46 @@ import "service/verify.proto";
 // Type of the object payload content
 enum ObjectType {
   // Just a normal object
-  REGULAR = 0;
+  OBJECT_TYPE_UNSPECIFIED = 0;
+
   // Used internally to identify deleted objects
-  TOMBSTONE = 1;
+  OBJECT_TYPE_TOMBSTONE = 1;
+
   // Identifies that the object holds StorageGroup information
-  STORAGE_GROUP = 2;
+  OBJECT_TYPE_STORAGE_GROUP = 2;
```

Comment:
Autogenerated code have enum prefix in these constant. `ObjectType_OBJECT_TYPE_STORAGE_GROUP` will look ugly.

Reply:
Agree

Reply:
Agree. Changed for all enums except default zero-valued first UNSPECIFIED fields. They must be unique package-wide.